cmake/configure.cmake

```diff
@@ -50,7 +50,11 @@ if(NOT WITH_PROFILER)
 endif(NOT WITH_PROFILER)
 
 if(NOT CMAKE_CROSSCOMPILING)
-    if(WITH_AVX AND AVX_FOUND)
+    if(WITH_AVX AND AVX512F_FOUND)
```

Comment:
We can not enable AVX512 yet, since it would cause the illegal instruction issue.

Reply:
OK, got it! It can be the reference when you solve the illegal instruction issue.